### DISCUSSION TITLE
Join the team of Contributors & Maintainers!

### DISCUSSION THREAD

**Discussion by connortann** [shap/shap#3559] [General]
This package is currently maintained by a rather small group, and we are keen to grow the team!

We are a friendly and open group, who aspire to collectively bring this package to a high technical standard. Our immediate priority is to address technical debt by fixing bugs and improving the test suite.

We would like to grow the team of maintainers to at least 6 or so, so that we have some comfortable redundancy and the package is never again left in an unmaintained state.

## A bit of history
 
This package was originally authored by Scott Lundberg, and has been a hugely popular part of the open source ecosystem for ML interpretability for many years. However, as of ~2022 the package was largely unmaintained, with no PRs accepted in over a year and several failing tests.
 
A "SHAP community fork" was started as a community-led effort (see #2942) to fix the issues over a number of months. In June 2023, the main collaborators of the fork were given access to the main repository and the fork was successfully merged into the main repo in #2943.

The repo is now maintained by a very small group, and we're eager for others to join.

## How to get involved
 
Contributions of all forms are welcome:

* Users can contribute by submitting [reproducible bug reports](https://shap.readthedocs.io/en/latest/contributing.html#writing-helpful-bug-reports) or feature requests on the issue tracker, or discussing ideas and questions in the GitHub discussion forums,
* Contributors can submit fixes and improvements via [Pull Requests](https://shap.readthedocs.io/en/latest/contributing.html#pull-requests-prs), and/or help reviewing other PRs
* Maintainers have triage or commit access to the project, and are typically people who have a track record of contributions to the project over a sustained period of time. If you would be interested in becoming a maintainer, please let us know e.g. by tagging an existing maintainer, or commenting below.
 
Please see the [Contributing Guide](https://shap.readthedocs.io/en/latest/contributing.html) for guidance on contributing and setting up a development environment.

If you are looking for a good place to get started, look for issues with the [good first issue](https://github.com/shap/shap/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label.

**Comment by Changchang001** [comment]
I'm using xgboost version: 2.1.1 and shap version: 0.42.1, running # Create TreeExplainer
explainer = shap.TreeExplainer(model_optimized)
shap_values = explainer.shap_values(X_test)
shap_interaction_values = explainer.shap_interaction_values(X_test) with the following problem UnicodeDecodeError: 'utf-8' codec can't decode byte 0x80 in position 390: invalid start byte.

**Comment by tupui** [comment]
Great to see that 👍

I don't have much bandwidth these days to take an active part. Still, feel free to ping me for topics related to sampling and sensitivity analysis. Or else if you think I can give some inputs.

**Reply by tylerjereddy** [reply]
Yeah, good to hear. We have some internal issues with SHAP in some virology/bioinformatics code and I've tried to encourage my postdocs to submit patches now and then. In any case, I'll try to do what I can on my end, and good that more stats-oriented upstream maintainers like Pamphile and Matt may occasionally chime in.
